tests/qiskit_qir/test_basic_gates.py

```
@@ -0,0 +1,206 @@
+# Two qubit gates mapping: qiskit_name -> qir_name
+TWO_QUBIT_GATES = {
+    "cx": "cnot",
+    "cz": "cz",
+    "swap": "swap",
+}
+
+
+class TestSingleQubitGates:
+    """Tests for single qubit gate conversions."""
+
+    @pytest.mark.parametrize("gate_name,qir_name", SINGLE_QUBIT_GATES.items())
```

Ruff (0.15.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)